CHANGELOG.md

```diff
@@ -7,3 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ## [1.0.0] - 2017-09-01
+
+## [1.0.1] - 2018-10-30
```

Comment:
We haven't done a 1.0.1 release yet so this shouldn't be here. This line should go up under the unreleased section. See https://keepachangelog.com/en/1.0.0/ for how this file should be formatted.